cli/lib/kontena/cli/master/token/create_command.rb

```diff
@@ -15,15 +15,17 @@ class CreateCommand < Kontena::Command
     option ['-u', '--user'], '[EMAIL]', 'Generate a token for another user'
     option ['--id'], :flag, "Only output the token ID"
     option ['--token'], :flag, "Only output the access_token (or authorization code)"
+    option ['-d', '--description'], '[DESCRIPTION]', 'Token description'
```

Comment:
Just noticed that here we use `[DESCRIPTION]` (with brackets) and somewhere else without brackets.. maybe we should unify these?

Reply:
Yes it seems to be  quite half and half. Functionally I don't think there's any difference but semantically it looks like the value would be optional and without it it would work like a flag.

Reply:
Actually we do have some required options like:

```
$ kontena certificate import
[ERROR] --private-key is required
```

In that case, the help should be (and is):

```
--private-key PATH
```

In cases where the option is optional, maybe it should say:

```
[ -d, --description DESCRIPTION] Token description
```

instead of the current:

```
-d, --description [DESCRIPTION] Token description
```